newIDE/app/src/GameDashboard/GamesList.js

```diff
@@ -44,19 +47,99 @@ export const GamesList = ({
         setError(null);
         const games = await getGames(getAuthorizationHeader, firebaseUser.uid);
         setGames(games);
-        // If a game id was passed, open it.
-        if (initialGameId) {
+      } catch (error) {
+        console.error('Error while loading user games.', error);
+        setError(error);
+      }
+    },
+    [authenticated, firebaseUser, getAuthorizationHeader]
+  );
+
+  const onRegisterGame = React.useCallback(
+    async () => {
+      if (!profile || !project) return;
+
+      const { id } = profile;
+      try {
+        setIsGameRegistering(true);
+        await registerGame(getAuthorizationHeader, id, {
+          gameId: project.getProjectUuid(),
+          authorName: project.getAuthor() || 'Unspecified publisher',
+          gameName: project.getName() || 'Untitled game',
```

Comment:
to translate? Same as for `Unspecified publisher`

Reply:
Not sure we need to translate it, we've used this logic elsewhere as well.
(+ it should not happen, the values are defined when creating a game, this is to please flow)